chandra_aca/tests/test_star_probs.py

```diff
@@ -581,6 +583,63 @@ def test_default_acq_prob_model():
     assert np.all(probs1 == probs2)
 
 
+def test_get_default_acq_prob_model_info_grid(monkeypatch):
+    monkeypatch.setenv("CHANDRA_MODELS_DEFAULT_VERSION", "3.48")
+    with conf.set_temp("default_model", "grid-*"):
+        info = get_default_acq_prob_model_info()
+
+    del info["data_file_path"]
+    del info["repo_path"]
+    del info["call_args"]["read_func"]
+    del info["call_args"]["read_func_kwargs"]
+
+    exp = {
+        "default_model": "grid-*",
+        "call_args": {
+            "file_path": "chandra_models/aca_acq_prob",
+            "version": None,
+            "repo_path": "None",
+            "require_latest_version": False,
+            "timeout": 5,
+        },
+        "version": "3.48",
+        "commit": "68a58099a9b51bef52ef14fbd0f1971f950e6ba3",
+        "md5": "3a47774392beeca2921b705e137338f4",
+    }
+    for name in chandra_models.ENV_VAR_NAMES:
+        exp[name] = os.environ.get(name)
```

Comment:
I don't actually understand what this is doing.  Are you just setting the "expected" values of these vars to match whatever is in os.environ to make the comparison easier in the next line?  If there is no true expected value it seems better not to compare them.